validatorPassword.js

```diff
@@ -0,0 +1,68 @@
+// Must be a minimum of 8 characters
+function validatePassword (password) {
+    const lowerCase = 'abcdefghijklmnopqrstuvwxyz'
+    const upperLetter = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    const numeric = '123456789'
+    const specialCharacter = '!@#$%^&*=+'
+    let isLongEnough = false
+    let hasLowerCase = false
+    let hasUpperCase = false
+    let hasNumeric = false
+    let hasSpecialCharacter = false
+    for (let i =0; i < password.length; i++) {  
+    if (password.length >= 8) isLongEnough = true
+    if (lowerCase.includes(password[i])) hasLowerCase = true
+    if (upperLetter.includes(password[i])) isLongEnough = true
+    if (numeric.includes(password[i])) hasNumeric = true
+    if (specialCharacter.includes(password[i])) hasSpecialCharacter = true
+          }
+          return isLongEnough && hasLowerCase && hasUpperCase && hasNumeric && hasSpecialCharacter
+        }   
+    
+    
+   // module.exports = validatorPassword
+    
+       console.log(validatePassword([true, false]))
+
```

Comment:
I uncommented the function however it is not the correct name it should be validatePassword  not validatorPassword